process/generators.py

```diff
@@ -44,23 +54,27 @@ def gen_units(proj):
 
 
 def nv_geom(proj):
-    if proj.field('the_geom') != '':
-        return [NameValue('geom', proj.field('the_geom'), 'planning')]
+    if proj.field('the_geom', PPTS.NAME) != '':
+        return [OutputNameValue('geom',
+                                proj.field('the_geom', PPTS.NAME),
+                                'planning')]
 
     return []
 
 
 def nv_all_units(proj):
     # TODO: more useful when we have more than just one data source
     result = []
-    if proj.field('market_rate_units_net'):
-        result.append(NameValue('net_num_units',
-                                proj.field('market_rate_units_net'),
-                                'planning'))
-    if proj.field('affordable_units_net'):
-        result.append(NameValue('net_num_units_bmr',
-                                proj.field('affordable_units_net'),
-                                'planning'))
+    if proj.field('market_rate_units_net', PPTS.NAME):
+        result.append(OutputNameValue(
+            'net_num_units',
```

Comment:
Unrelated to this change (so feel free to just address later with a different pr) but feels like this should be net_num_units_market_rate

Reply:
it's not, though. the original field name is the one that needs fixing i think, because "market_rate_units_net" in the schema-less really means "all units", no?

Reply:
Yeah I think you're right, just need to fix the inconsistency but not for this pr anyway, resolving